### ISSUE TITLE
Removed Firefox Addon Question => not-on-amo/last-visit-search

### ISSUE THREAD

**Issue by Hammerfest** [igorlogius/igorlogius#5]
With increasing age and memory issues, I was looking for an add-on to display thumbnails of visited sites.

I came across a thread about your Visual History add-on.

Was that removed for a specific reason?
Is there something better/equal?

**Comment by igorlogius** [comment]
> I came across a thread about your Visual History add-on.

I assume you mean 
https://github.com/igorlogius/not-on-amo/blob/main/last_visit_search-1.18.9.xpi

> Was that removed for a specific reason?

It should still work but it has been reported to me that with time and growing history it can get rather sluggish/unresponsive and since i kind of did not have a good solution for the problem at the time i thought it would be better to take it down. Obviously this issue depends on your usage and available system resources.  

> Is there something better/equal?

Not sure. Havent looked into these kinds of addons for a while, sorry.

**Comment by Hammerfest** [comment]
That is probably it, I found the old link and wayback capture from:
https://addons.mozilla.org/en-US/firefox/addon/visualhistory/

But it only lists 0.2.0 so probably in its early days.

Will give it a go, I did/do expect it to use more resources, but RAM/SSD space is not something im worried about compared to when I used to use chrome and it would slag my system!

Is there a way to limit/cull thumbnail history? Like setting a limit of only thumbnailing the last week/month/custom?
